app/integration_action_test.go

```diff
@@ -481,6 +512,35 @@ func TestPostActionRelativeURL(t *testing.T) {
 	}))
 	defer ts.Close()
 
+	setupPluginApiTest(t,
```

Comment:
Can we duplicate `TestPostActionRelativeURL` as `TestPostActionRelativePluginURL`, and have the former exclusively use the `httptest` server, and the latter exclusively the `setupPluginApiTest`? As it stands right now, if you rip out your new code, the test still "passes" because it ends up hitting the test server instead.

Reply:
I made the requested changes however had to invert some of the logic assertions after `DoPostAction` called in `TestPostActionRelativeURL` because requests with relative URLs prefixed with `/plugins/` or `plugins/` skip the call to the network and don't call the test server.

Reply:
condensed the tests in `TestPostActionRelativePluginURL`